doc/GoogleLanguageConnector.md

```diff
@@ -0,0 +1,50 @@
+# reshuffle-google-language-connector
+
+### Reshuffle GoogleLanguageConnector
+
+This connector provides an Interface to the Google Language Client.
+
+#### Configuration Options:
+```typescript
+interface GoogleLanguageConnectorConfigOptions {
+  credentials: string
+}
+```
+#### Connector actions
+
+##### analyzeSentiment
+Returns the analyzed sentiment response
+```typescript
+analyzeSentiment(
+    text: string,
+  ): Promise<protos.google.cloud.language.v1.IAnalyzeSentimentResponse>
+```
+
+Analyzes the sentiment of the provided text
+
+###### Example
+```js
+const connector = new GoogleLanguageConnector({ credentials })
+const result = await connector.analyzeSentiment("The text I want analzed")
+console.log("Language:", result.language, " Sentiment score: ", result.documentSentiment.score)
+```
+
+##### sdk
+Returns the Google Language Service Client
+```typescript
+sdk(): v1.LanguageServiceClient
+```
+
```

Comment:
I'd add:

See documentation in github: <link to the github library you use for documentation>